hibike/devices/BatteryBuzzer/eeprom.cpp

```diff
@@ -0,0 +1,145 @@
+
+#include "eeprom.h"
+
+//low-level functions that deal with storing the calibration values using the agreed upon format.
+
+float get_calibration()  //returns calibration voltage.  Returns -1 (float) if no calibration.
+{
+   if(EEPROM.read(0)=='C' && EEPROM.read(1)=='A' && EEPROM.read(2)=='L' && EEPROM.read(3)==':')
+   {
+      //instantiate a float.
+      float f = 0.00000f;
+      EEPROM.get(4,f); //calibration value is stored in location 4
+      return f;
+   }
+   else
+   {
+      return -1.0;
+   }
+
+
+}
+
+void update_triple_calibration() //updates the global "calib" array based on EEPROM.
+//Calib will contain datasheet values (2.56) if there's no calibration.
+//If there is, the appopriate elements will be updated.
+{
+
+  
+  int next_addr = 0;
+  for(int i = 1; i<=3; i++)
+  {
+     if(EEPROM.read(next_addr)=='C' && EEPROM.read(next_addr+1)=='A' && EEPROM.read(next_addr+2)=='L' && EEPROM.read(next_addr+3) == char(i+48) &&EEPROM.read(next_addr+4)==':')
```

Comment:
Writing `48` as an integer is not strictly necessary (see my other comment on this issue)